CruzHacks/cruzhacks-2021-website
# Speakers Section

Problem
=======
As a hacker, 
i want to see who the keynote speakers are 
so I can gauge my interest in the event

[Link to Asana Ticket](https://app.asana.com/0/1197738625416321/1199382769817329/f)

Solution
========
Created a `Background` component that contains the constellations and laid the cards on top of that layer in a flex container.  

Change Summary: 
-------------------
* added speaker component card 99ce803
* added pictures and svgs for backdrop 7983f6b
* added background component for speakers section b8aaa89
* added speakers component and styling 039d4eb
* rearranged imports and added speakers and props components cc396ad
* added newlines to the json prop files 5cb2d59

Steps to Verify:
----------------
1. Checkout this branch and start a local development environment 
2. Verify that the cards all appear in the speaker section of the page. 
3. Verify that in the initial view, there are 3 cards in each row and that it is responsive to mobile. 

Screenshots (optional):
-----------------------
<img width="2551" alt="Screen Shot 2021-01-02 at 1 08 55 AM" src="https://user-images.githubusercontent.com/40477553/103454265-191c7480-4c97-11eb-9879-50762e458d58.png">

**NOTE:** I think we might want to take a look at the `Background` component of the entire site to ensure that this section lands in the blue section of the webpage so the constellations don't look out of place.
